ecommerce/api.py

```diff
@@ -242,18 +243,26 @@ def refund_order(*, order_id: int, **kwargs):
     unenroll = kwargs.get("unenroll", False)
 
     with transaction.atomic():
-
-        order = FulfilledOrder.objects.select_for_update().get(id=order_id)
-
+        if reference_number is not None:
+            order = FulfilledOrder.objects.select_for_update().get(
+                reference_number=reference_number
+            )
+        elif order_id is not None:
+            order = FulfilledOrder.objects.select_for_update().get(pk=order_id)
+        else:
+            log.debug(
+                f"Either order_id or reference_number is required to fetch the Order. "
```

Comment:
Doesn't need to be an f-string and has an extra trailing space. This should also probably be an error.